R/combineSNV_CNA.R

```diff
@@ -0,0 +1,73 @@
+# combineSNV_CNA.R
+
+#' Combine mutation data from CNA and SNV
+#'
+#' \code{combineSNV_CNA} loads the vector of preprocessed dataset from filtered CNA or SNV files,
+#'  creates hash keys corresponding to the input file type, combine and save the hashed
```

Comment:
That these are hashed is actually only of interest to us - the user feeds text files and gets a data frame.